actiontext/lib/action_text/attachment.rb

```diff
@@ -3,6 +3,17 @@
 require "active_support/core_ext/object/try"
 
 module ActionText
+  # = Action Text \Attachment
+  #
+  # Attachments serialize attachables to HTML or plain text.
+  #
+  #   class Person < ApplicationRecord
+  #     include ActionText::Attachable
+  #   end
+  #
+  #   attachable = Person.create! name: "Javan"
+  #   attachment = ActionText::Attachment.from_attachable(attachable)
+  #   attachment.to_html # => "<action-text-attachment sgid=\"BAh7CEk...
```

Comment:
Might make sense to just add the closing quote on this example? (Same below)
```suggestion
  #   attachment.to_html # => "<action-text-attachment sgid=\"BAh7CEk..."
```

Reply:
Thanks @adrianna-chang-shopify ! I think that makes sense.